app/views/subscriptions/unsubscribe.html.haml

```diff
@@ -9,5 +9,20 @@
       .govuk-panel__body
         %p= t('.confirmation')
 
-  .govuk-grid-column-two-thirds
-    %p.govuk-body=t('.resubscribe_html', link: link_to(t('.resubscribe_link_text'), new_subscription_path(search_criteria: @subscription.search_criteria_to_h), class: 'govuk-link', id: 'job-alert-link-from-cancel-unsubscribe-gtm'))
+    = form_for @unsubscribe_feedback_form, url: unsubscribe_feedback_subscription_path(@subscription) do |f|
+      = f.govuk_error_summary
+
+      %h2.govuk-heading-l
+        = t('.heading')
+
+      = f.govuk_radio_buttons_fieldset(:reason, legend: { size: 'm' }) do
+        - UnsubscribeFeedback.reasons.keys.each do |reason|
+          - if reason == "other_reason"
+            = f.govuk_radio_button :reason, reason do
+              = f.govuk_text_field :other_reason, label: { size: 's' }
+          - else
+            = f.govuk_radio_button :reason, reason
+
+      = f.govuk_text_area :additional_info, label: { size: 's' }, rows: 5, max_chars: 1200
```

Comment:
Does the max chars GOVUK JS count work automatically here? You might need to add some classes and a specific id (like in `job_details.html.haml`)

Reply:
It works automatically

Reply:
Cool, we should be able to remove the extra code in `app/views/hiring_staff/vacancies/build/job_details.html.haml` and `app/views/hiring_staff/vacancies/copy/new.html.haml` too!